<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NiippControlWidget</class>
 <widget class="QWidget" name="NiippControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_mode">
       <property name="text">
        <string>Режим работы</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_antena">
       <property name="text">
        <string>Выбор антенны</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="_cb_antena"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="_cb_mode"/>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="_pb_start">
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="_le_distance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QSpinBox" name="_sb_power"/>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLineEdit" name="_le_status"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbl_power">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Зона</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="lbl_l1">
       <property name="text">
        <string>Широта</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lbl_l2">
       <property name="text">
        <string>Долгота</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="_clear_uvod">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="_le_lat"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_le_lon"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QPushButton" name="_pb_enable_complex">
     <property name="text">
      <string>Включить комплекс</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSlider" name="_sl_power">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lbl_status">
     <property name="text">
      <string>Статус</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lbl_distance">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Радиус</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
